waffle-chai/test/contracts/Events.ts

```diff
@@ -0,0 +1,53 @@
+export const EVENTS_SOURCE = `
```

Comment:
Good idea to simplify tests

Reply:
Yeah, I was sick of everything being compiled every time and needing all of the dependencies